### ISSUE TITLE
[Example] support more TFLite image classification models

### ISSUE THREAD

**Issue by huningxin** [intel/webml-polyfill#220]
The existing example (https://github.com/intel/webml-polyfill/tree/master/examples/mobilenet) only supports TFLite MobileNet model for image classification. This task is to make the example more general to support more TFLite image classification models hosted here:

https://github.com/tensorflow/tensorflow/blob/master/tensorflow/contrib/lite/g3doc/models.md#image-classification-float-models

It requires:
1. refactor the example to load, compile and execute multiple models
2. identify the unsupported ops if the model is not supported

**Comment by Wenzhao-Xiang** [comment]
These are the models that are supported with present ops:
**Mobilenet_V1 / Mobilenet_V2 / Inception_V3 / Squeezenet**

And these are the models that are unsupported now and the ops they need:

model |    FullyConnected | Mean | Pad | Split | StridedSlice
-- | -- | -- | -- | -- | --
Densenet |   √ | √ |   |   |  
Inception_V4 |   √ |   |   |   |  
Inception_Resnet_V2 |    √ |   |   |   |  
Nasnet_Mobile |    √ | √ | √ | √ | √
Nasnet_Large |    √ | √ | √ | √ | √
ResNet_V2_50 |      | √ | √ |   |  
ResNet_V2_101 |      | √ | √ |   |  
MnasNet_1.0_224 |    √ | √ |   |   |

**Comment by Wenzhao-Xiang** [comment]
NNAPI now support **[FullyConnected](https://developer.android.com/ndk/reference/group/neural-networks#group___neural_networks_1ggaabbe492c60331b13038e39d4207940e0aaada7a3dbaf4676aba560c933ff610c5)** / **[Mean](https://developer.android.com/ndk/reference/group/neural-networks#group___neural_networks_1ggaabbe492c60331b13038e39d4207940e0a047fe95a35b27f45c05432b6ca18eb6c)** / **[Pad](https://developer.android.com/ndk/reference/group/neural-networks#group___neural_networks_1ggaabbe492c60331b13038e39d4207940e0aaced01fc41e401b81cefcf53780558d1)** / **[StridedSlice](https://developer.android.com/ndk/reference/group/neural-networks#group___neural_networks_1ggaabbe492c60331b13038e39d4207940e0a89695302f8b1e7ae7ce8f4d8c0b8a752)**.
And this is the definition of **[Split](https://js.tensorflow.org/api/0.12.0/#split)** in tensorflow.js.

**Comment by Wenzhao-Xiang** [comment]
This is the tflite model for Image Classification and the ops they include.

Model \ Op Type | ADD | MUL | CONV_2D | DEPTHWISE_CONV_2D | AVERAGE_POOL_2D | MAX_POOL_2D | CONCATENATION | RESHAPE | SOFTMAX | SQUEEZE | FULLY_CONNECTED | Mean | PAD | SPLIT | STRIDED_SLICE | RELU
-- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | --
Mobilenet_V1 |   |   | √ | √ | √ |   |   |   | √ | √ |   |   |   |   |   |  
Mobilenet_V2 | √ |   | √ | √ | √ |   |   | √ | √ |   |   |   |   |   |   |  
SqueezeNet |   |   | √ |   | √ | √ | √ | √ | √ |   |   |   |   |   |   |  
Densenet | √ | √ | √ |   | √ | √ | √ |   | √ |   | √ | √ |   |   |   |  
Inception_V3 |   |   | √ |   | √ | √ | √ | √ | √ |   |   |   |   |   |   |  
Inception_V4 |   |   | √ |   | √ | √ | √ |   | √ |   | √ |   |   |   |   |  
Inception_Resnet_V2 | √ |   | √ |   | √ | √ | √ |   |   |   | √ |   |   |   |   |  
Nasnet_Mobile | √ | √ | √ | √ | √ | √ | √ |   | √ |   | √ | √ | √ | √ | √ | √
Nasnet_Large | √ | √ | √ | √ | √ | √ | √ |   | √ |   | √ | √ | √ | √ | √ | √
ResNet_V2_50 | √ | √ | √ |   |   | √ |   | √ | √ |   |   | √ | √ |   |   |  
ResNet_V2_101 | √ | √ | √ |   |   | √ |   | √ | √ |   |   | √ | √ |   |   |  
MnasNet_1.0_224 | √ |   | √ | √ |   |   |   |   |   |   | √ | √ |   |   |   |  

**And Here is the [Wiki page](https://github.com/intel/webml-polyfill/wiki/Tflite-Model-for-Image-Classification-and-the-Ops-They-Include)**
